### ISSUE TITLE
A11yhelp dialog should be accessible by a toolbar button

### ISSUE THREAD

**Issue by Comandeer** [ckeditor/ckeditor4#4968]
## Type of report

Bug

## Provide detailed reproduction steps (if any)

1. Open a demo with the editor.

### Expected result

A11yhelp dialog is accessible via means other than keystroke, e.g. toolbar button.

### Actual result

The dialog is accessible only via a keystroke.

## Other details

* Browser: any
* OS: any
* CKEditor version: 4.5.0+
* Installed CKEditor plugins: a11yhelp
